src/main.js

```diff
@@ -25,32 +40,51 @@ function getOctokit() {
     throw new Error("Failed to get a proper GitHub client.");
   }
 }
+
+function validateEmail({ email, emailRegex }) {
+  return new RegExp(emailRegex).test(email);
+}
+
+function validateCreatedUser({ issue, createdUserRegex }) {
```

Comment:
I found the function name very confusing. Since we validating that the user is a trusted user I think the name should change to:

```suggestion
function isTrustedUser({ issue, trustedUserRegex }) {
```